internal/manifests/config/configmap.go

```diff
@@ -22,15 +25,15 @@ type S3 struct {
 	Bucket   string
 }
 
-func BuildConfigMap(tempo v1alpha1.Microservices, params Params) (*corev1.ConfigMap, error) {
+func BuildConfigMap(tempo v1alpha1.Microservices, params Params) (*corev1.ConfigMap, string, error) {
```

Comment:
Maybe we could put a comment on top that describes whats actually returned. wdyt?

Reply:
Good idea, done.